src/model-interfaces.ts

```diff
@@ -32,6 +32,7 @@ export enum Sorteringsfelt {
     MOTER_IDAG = 'moterMedNAVIdag',
     VEDTAKSTATUS = 'vedtakstatus',
     VEDTAKSTATUS_ENDRET = 'vedtak_status_endret',
+    ANSVARLIG_VEILEDER_FOR_VEDTAK = 'ansvarligveilederforvedtak',
```

Comment:
ansvarligveilederforvedtak -> ansvarlig_veileder_for_vedtak
can we try to use _one_ convention here?

Reply:
Yep, I agree..